content/swiftui/concepts/gestures/gestures.md

```diff
@@ -0,0 +1,137 @@
+---
+Title: 'Gestures' 
+Description: 'Defines interactions from taps, clicks, and swipes to fine-grained gestures.' 

```

Comment:
Terms should starts with a verb, but it's not necessary. I did notice similarities between this and Apple Docs so if we can have this match the introduction that starts with "Gestures are..." that would be 👍